app/vmui/packages/vmui/src/pages/ExploreLogs/ExploreLogsBarChart/ExploreLogsBarChart.tsx

```diff
@@ -31,7 +31,18 @@ const ExploreLogsBarChart: FC<Props> = ({ logHits, error, loaded }) => {
     return [xAxis, yAxis] as AlignedData;
   }, [logHits]);
 
-  const noData = data.some(d => d.length === 0);
+  const noDataMessage: string = useMemo(() => {
+    const noData = data.every(d => d.length === 0);
+    const noTimestamps = data[0].length === 0;
+    const noValues = data[1].length === 0;
+    if (noData) {
+      return "No logs volume available\nNo volume information available for the current queries and time range.";
+    } else if (noTimestamps) {
```

Comment:
nit: else is redundant

Reply:
In cases of aggregation, the API may return a value without timestamps; in this case, the message will explicitly indicate this.